elpis/endpoints/transcription.py

```diff
@@ -59,15 +59,27 @@ def status():
 @bp.route("/text", methods=['GET'])
 def text():
     transcription: Transcription = app.config['CURRENT_TRANSCRIPTION']
-    # TODO fix this to return json wrapper
-    return transcription.text()
+    data = {
```

Comment:
Previously, the endpoint was passing file contents back as the request data. This wraps the file contents in a data object with status etc.